auditlog/models/rule.py

```diff
@@ -364,11 +472,20 @@ def create_logs(self, uid, res_model, res_ids, method,
         http_session_model = self.env['auditlog.http.session']
         for res_id in res_ids:
             model_model = self.env[res_model]
+            # Do extra check for active_model situations where res_model
+            # is not preloaded in auditlog model_cache
+            model_id = self.pool._auditlog_model_cache.get(res_model, False)
```

Comment:
Yes, I fell for the trap that get()'s default is evaluated immediately. +1 for your suggestion.